pymobiledevice3/services/webinspector.py

```diff
@@ -313,11 +331,14 @@ async def _wait_for_page(self, session_id):
                         return page
             await asyncio.sleep(0)
 
-    async def _wait_for_application(self, bundle='', app_id=''):
+    async def _wait_for_application(self, bundle: str = '', app_id: str = '', timeout: Union[float, int] = 3):
```

Comment:
Don't implement the timeout, the call should use `asyncio.wait_for`

Reply:
done